.travis.yml

```diff
@@ -46,4 +46,4 @@ before_script:
   - composer validate
 
 script:
-  - behat --format progress --strict
+  - bash bin/test.sh
```

Comment:
> Is the bin/test.sh file executable (has its executable bit set)?

Yes.

> If this is the case, wouldn't it be preferable to not reference bash separately, but to just run bin/test.sh? It has a shebang that calls bash, so Travis should be able to figure it out from there (at least I assume so).

I used `bash` for parity with the `bash bin/install-package-tests.sh` call above.

Functionally, it doesn't make a difference.

Personally, I don't have a strong preference, but I think `bash` is a bit more literal for the end user.